.github/workflows/build-with-bal-test-graalvm.yml

```
@@ -0,0 +1,39 @@
+name: GraalVM Check
+
+on:
+    workflow_dispatch:
+        inputs:
+            lang_tag:
+                description: Branch/Release Tag of the Ballerina Lang
+                required: true
+                default: master
+            lang_version:
+                description: Ballerina Lang Version (If given ballerina lang build will be skipped)
+                required: false
+                default: ''
+            native_image_options:
+                description: Default native-image options
+                required: false
+                default: ''
+    schedule:
+        -   cron: '30 18 * * *'
+    pull_request:
+        branches:
+            - master
```

The default branch should be `main`. Please check all the places.

```suggestion
            - main
```